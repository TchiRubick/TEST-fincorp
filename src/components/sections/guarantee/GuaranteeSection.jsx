import React from 'react';

import './GuaranteeSection.scss';

const GuaranteeSection = () => (
  <div className="section-guarantee">
    <div className="section-guarantee__block">
      Blocl card
    </div>
  </div>
);

export default GuaranteeSection;
