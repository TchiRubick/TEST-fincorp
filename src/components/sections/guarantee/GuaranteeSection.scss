@import '../../style.scss';

.section-guarantee {
  margin: 0.5rem auto;

  .section-guarantee__block {
    border-radius: 0.4rem;
    box-shadow: 0 4px 4px 0 rgba(0,0,0,0.1);
    padding: 0.5rem;
    width: 50rem;
    margin: auto;

    display: flex;
    justify-content: space-between;

    .section-guarantee__block-text {
      display: flex;
      flex-direction: column;
      align-self: center;
      
      .section-guarantee__block-text__title {
        @include title;
      }

      .section-guarantee__block-text__description {
        @include sub-text;
      }
    }

    .section-guarantee__block-text__button {
      align-self: center;
      height: 3rem;
    }
  }
}