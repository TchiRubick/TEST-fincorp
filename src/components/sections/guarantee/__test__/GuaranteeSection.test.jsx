import React from 'react';
import ReactDom from 'react-dom';
import { cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import GuaranteeSection from '../GuaranteeSection';

afterEach(cleanup);

it('renders good', () => {
  const div = document.createElement('div');

  ReactDom.render(<GuaranteeSection />, div);
});

it('as expected snapshot 1', () => {
  const tree = renderer.create(<GuaranteeSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
