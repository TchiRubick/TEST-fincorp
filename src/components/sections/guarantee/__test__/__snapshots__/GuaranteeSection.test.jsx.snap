// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`as expected snapshot 1 1`] = `
<div
  className="comp-container"
>
  <div
    className="section-guarantee"
  >
    <div
      className="section-guarantee__block"
    >
      <img
        alt="shield"
        src="shield-2.png"
      />
      <div
        className="section-guarantee__block-text"
      >
        <span
          className="section-guarantee__block-text__title"
        >
          100% Satisfaction Guarantee
        </span>
        <span
          className="section-guarantee__block-text__description"
        >
          We offer no questions asked refund policy for 14 days from the policy date.
        </span>
      </div>
      <div
        className="section-guarantee__block-text__button"
      >
        <button
          className="comp-button-base __green"
          type="button"
        >
          Get your free Quote
        </button>
      </div>
    </div>
  </div>
</div>
`;
